### ISSUE TITLE
[BUG] Unable to access settings page

### ISSUE THREAD

**Issue by proffalken** [thanksmister/wallpanel-android#292]
**Describe the bug**

Hello, first of all thanks for an awesome project, it's really working well with my Hubitat dashboard!

At some point I've managed to hide the settings button and I've no idea how.  I've searched through the issues on here and seen that apparently I can get into the settings by pressing the "back" button, however that doesn't appear to work on my tablet.

Is there any other way to get the "settings" button on the screen?

**To Reproduce**

I wish I could tell you, I've absolutely no idea, sorry  - I suspect it was user-error.

**Smartphone (please complete the following information):**
 - Device: VANKYO Z4
 - OS: Android 9
 - Application version: 0.9.5 build 6 (taken from the tablet settings page, not from within the app)

**Comment by thanksmister** [comment]
There is an option in the settings with a a warning that allows you to remove the settings button as well as options to make it invisible or change the corner.   

If you no longer have the settings button in any of the corners, you must have selected to remove it.

Once you remove the button, opening the settings must be called from MQTT, usually a button in your HA dashboard.


The only way to open the settings once the button is removed is to use MQTT command as outlined in the wiki or uninstall and install the application again.

**Comment by proffalken** [comment]
@thanksmister ok, that's annoying, I didn't setup the MQTT stuff either! :(

An uninstall/re-install doesn't seem to have solved it, but let me try again - this is a "disposable" install, so it's easy to wipe and start again if needed.

As I say, almost certainly "operator error" here, appreciate your time.

**Comment by proffalken** [comment]
@thanksmister FWIW, I've uninstalled and re-installed and it doesn't appear to have wiped the stored preferences on the tablet when it's done this.

The app has definitely gone and come back, but as soon as I launch it then the app returns and auto-loads the same dashboard that was configured previously.

Is there a particular folder I need to delete in order to wipe the config?

**Comment by proffalken** [comment]
OK, clearing the cache and storage from within Android itself resolved this, the settings button is now back.

Closing this :)

**Comment by thanksmister** [comment]
You probably have backup application data on for the application.
